docs/user-guide/rug/editors.md

```diff
@@ -126,45 +126,45 @@ public class ${this.controller_class_name}Controller {
 export const addSpringBootRestController = new AddSpringBootRestController();
 ```
 
-The first few lines group the Rug typings our script will be using throughout 
-(lines 1&ndash;4). Next, we declare our editor through TypeScript [decorators][] 
-(line 6). The first argument of the `#!typescript @Editor` decorator is the name 
-of the editor. This the public visible and discoverable name of the Rug. This 
-name, along with the editors group and repository, form the fully-qualified name 
-of the editor. The second argument of the `#!typescript @Editor` decorator is a 
-short description of the editor. The following line uses the 
-`#!typescript @Tags` decorator to apply some tags to our editor so people can 
-search for it more easily. Using the `#!typescript @Tags` decorator is optional 
+The first few lines group the Rug typings our script will be using throughout
+(lines 1&ndash;4). Next, we declare our editor through TypeScript [decorators][]
+(line 6). The first argument of the `#!typescript @Editor` decorator is the name
+of the editor. This the public visible and discoverable name of the Rug. This
+name, along with the editors group and repository, form the fully-qualified name
+of the editor. The second argument of the `#!typescript @Editor` decorator is a
+short description of the editor. The following line uses the
+`#!typescript @Tags` decorator to apply some tags to our editor so people can
+search for it more easily. Using the `#!typescript @Tags` decorator is optional
```

Comment:
really it means they can `find` it more easily - searching is actually hard right now with the bot! :)